README.md

```diff
@@ -1,4 +1,6 @@
 [![Build Status](https://travis-ci.org/skydive-project/skydive.png)](https://travis-ci.org/skydive-project/skydive)
+[![Go Report Card](https://goreportcard.com/badge/github.com/skydive-project/skydive)](https://goreportcard.com/report/github.com/skydive-project/skydive)
+[![Coverage Status](https://coveralls.io/repos/github/redhat-cip/skydive/badge.svg?branch=master)](https://coveralls.io/github/redhat-cip/skydive?branch=master)
```

Comment:
we are not using coveralls for the moment. moreover the URL seems outdated "redhat-cip"